src/codewhisperer/commands/basicCommands.ts

```diff
@@ -62,6 +62,19 @@ export const enableCodeSuggestions = Commands.declare(
     }
 )
 
+export const disableCodeSuggestions = Commands.declare(
+    'aws.codeWhisperer.disableCodeSuggestions',
+    (context: ExtContext) => async () => {
+        await set(CodeWhispererConstants.autoTriggerEnabledKey, false, context.extensionContext.globalState)
+        await vscode.commands.executeCommand('setContext', 'CODEWHISPERER_ENABLED', true)
```

Comment:
This command says "disable" but here it's set to true -- is this intended?

Reply:
revised